app/controllers/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UsersController < ApplicationController
+
+  def new
+  end
+
+  def create
+    user = User.new(user_params)
+    if user.save
```

Comment:
Technically I think that setting up the conditional here is outside what is needed for this user story, it should have just been user.save, then implement the conditional with sad path for user story 12

Reply:
Removed as well